data/sql/updates/pending_db_world/rev_1601843049417097700.sql

```diff
@@ -0,0 +1,25 @@
+(12916, 'Varos Cloudstrider and his ring guardians protect the second ring.  Your drakes are more than a match for the ring guardians, but Varos stands behind an impenetrable shield created from the energy of the Oculus itself.  Ten centrifuge constructs power the shield from the ring and platforms above.  Destroy them and Varos will be vulnerable.$B$BI can grant you the power to call upon a drake from the red flight.  Speak to Eternos or Verdisa if you prefer to draw on the power of the bronze or the green.', '', 26950, 0, 1, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, 12340);
+
+DELETE FROM `broadcast_text` WHERE `ID` IN (26950, 29536);
+INSERT INTO `broadcast_text` (`ID`, `Language`, `MaleText`, `FemaleText`, `EmoteID0`, `EmoteID1`, `EmoteID2`, `EmoteDelay0`, `EmoteDelay1`, `EmoteDelay2`, `SoundId`, `Unk1`, `Unk2`, `VerifiedBuild`) VALUES 
+(26950, 0, 'Varos Cloudstrider and his ring guardians protect the second ring.  Your drakes are more than a match for the ring guardians, but Varos stands behind an impenetrable shield created from the energy of the Oculus itself.  Ten centrifuge constructs power the shield from the ring and platforms above.  Destroy them and Varos will be vulnerable.$B$BI can grant you the power to call upon a drake from the red flight.  Speak to Eternos or Verdisa if you prefer to draw on the power of the bronze or the green.', '', 0, 0, 0, 0, 0, 0, 0, 0, 1, 18019),(29536, 0, 'Varos Cloudstrider and his ring guardians protect the second ring.  Your drakes are more than a match for the ring guardians, but Varos stands behind an impenetrable shield created from the energy of the Oculus itself.  Ten centrifuge constructs power the shield from the ring and platforms above.  Destroy them and Varos will be vulnerable.', 'Varos Cloudstrider and his ring guardians protect the second ring.  Your drakes are more than a match for the ring guardians, but Varos stands behind an impenetrable shield created from the energy of the Oculus itself.  Ten centrifuge constructs power the shield from the ring and platforms above.  Destroy them and Varos will be vulnerable.', 0, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
```

Comment:
Why two blanks after every period?

Reply:
This is how i found the text, only added breaks where they we re not, i will check how this text look on the npc vs retail and check to see if the spaces are needed or if they change anything.

Reply:
Cool =)

Reply:
Seems like it is the same formatting.